modules/platforms/dotnet/Apache.Ignite.Core/IDataRegionMetrics.cs

```diff
@@ -43,6 +43,11 @@ public interface IDataRegionMetrics
         /// </summary>
         long TotalAllocatedSize { get; }
 
+        /// <summary>
+        /// Gets a total number of pages used for storing the data.
```

Comment:
Please put here the same detailed comment that we have in `DataRegionMetrics.java`

Reply:
Done